src/web/components/ArticleHeadline.tsx

```diff
@@ -42,6 +54,11 @@ const lightFont = css`
     font-weight: 300;
     font-size: 2.125rem;
     line-height: 2.375rem;
+    ${until.mobileMedium} {
+        ${headline.small()};
+        font-weight: normal;
+        font-weight: 300;
+    }
```

Comment:
Does `headline.small({fontWeight: 'light'})` work here?

Reply:
It does. Thanks.

Not sure why font weight was being set twice there either